packages/cli/src/verify.ts

```diff
@@ -54,31 +53,42 @@ export async function verify({
 
   systems.map(({ name, bytecode }) =>
     verifyQueue.add(() =>
-      verifyContract(
-        {
-          name,
-          rpc,
-          verifier,
-          verifierUrl,
-          address: getCreate2Address({
-            from: deployerAddress,
-            bytecode: bytecode,
-            salt,
-          }),
-        },
-        { profile: foundryProfile },
-      ).catch((error) => {
+      verifyContract({
+        name,
+        rpc,
+        verifier,
+        verifierUrl,
+        address: getCreate2Address({
+          from: deployerAddress,
+          bytecode: bytecode,
+          salt,
+        }),
+      }).catch((error) => {
         console.error(`Error verifying system contract ${name}:`, error);
       }),
     ),
   );
 
-  Object.entries(
-    usesProxy ? getWorldProxyFactoryContracts(deployerAddress) : getWorldFactoryContracts(deployerAddress),
-  ).map(([name, { bytecode }]) =>
-    verifyQueue.add(() =>
-      verifyContract(
-        {
+  // If the verifier is Sourcify, attempt to verify MUD core contracts
+  // There are path issues with verifying Blockscout and Etherscan
+  if (verifier === "sourcify") {
+    // Install subdependencies so contracts can compile
```

Comment:
it's less about compiling and more about the dependent files existing right?

Reply:
well, the files have to exist in node_modules to be compiled